From: Alexandre Ghiti <alexandre.ghiti@canonical.com>
Date: Wed, 22 Sep 2021 06:38:14 +0200
Subject: include: Rename HiFive partition UUIDs

Both HiFive Unleashed and HiFive Unmatched bootloaders seek for the same
UUIDs to load the next stage bootloader: the current name makes partitions
on Unmatched board appear as 'Unleashed'.

Fix that by removing the 'Unleashed' part of the current naming so it
fits both.

Signed-off-by: Alexandre Ghiti <alexandre.ghiti@canonical.com>
---
 include/pt-gpt-partnames.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/pt-gpt-partnames.h b/include/pt-gpt-partnames.h
index 6c54a71..e2e0432 100644
--- a/include/pt-gpt-partnames.h
+++ b/include/pt-gpt-partnames.h
@@ -165,6 +165,6 @@ DEF_GUID("CEF5A9AD-73BC-4601-89F3-CDEEEEE321A1", N_("QNX6 file system")),
 /* Plan 9 */
 DEF_GUID("C91818F9-8025-47AF-89D2-F030D7000C2C", N_("Plan 9 partition")),
 
-/* HiFive Unleased bootloaders */
-DEF_GUID("5B193300-FC78-40CD-8002-E86C45580B47", N_("HiFive Unleashed FSBL")),
-DEF_GUID("2E54B353-1271-4842-806F-E436D6AF6985", N_("HiFive Unleashed BBL")),
+/* HiFive bootloaders */
+DEF_GUID("5B193300-FC78-40CD-8002-E86C45580B47", N_("HiFive FSBL")),
+DEF_GUID("2E54B353-1271-4842-806F-E436D6AF6985", N_("HiFive BBL")),
